rllib/core/learner/torch/torch_differentiable_learner.py

```diff
@@ -255,6 +264,28 @@ def _make_functional_call(
             return_dict=True,
         )
 
+    @override(DifferentiableLearner)
```

Comment:
same here: Why do we copy all this code? Let's just do `return TorchLearner._get_tensor_variable(...)`.

Reply:
All not possible, because we cannot derive from `TorchLearner` as that class has methods without parameters and the its methods like `_get_tensor_variable` are not static.